server/src/main/java/org/elasticsearch/ElasticsearchException.java

```diff
@@ -1006,8 +1007,8 @@ public String toString() {
             UNKNOWN_VERSION_ADDED),
         TYPE_MISSING_EXCEPTION(org.elasticsearch.indices.TypeMissingException.class,
                 org.elasticsearch.indices.TypeMissingException::new, 137, UNKNOWN_VERSION_ADDED),
-        FAILED_TO_COMMIT_CLUSTER_STATE_EXCEPTION(org.elasticsearch.discovery.Discovery.FailedToCommitClusterStateException.class,
-                org.elasticsearch.discovery.Discovery.FailedToCommitClusterStateException::new, 140, UNKNOWN_VERSION_ADDED),
+        FAILED_TO_COMMIT_CLUSTER_STATE_EXCEPTION(FailedToCommitClusterStateException.class,
```

Comment:
Almost all of these registrations use the fully-qualified class name (except `CoordinationStateRejectedException`, oops) so it looks like this should too.

Reply:
right, also fixed for `CoordinationStateRejectedException` in d3c5a3d2918